nodeup/pkg/model/kube_apiserver.go

```diff
@@ -82,17 +81,13 @@ func (b *KubeAPIServerBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
-	// Touch log file, so that docker doesn't create a directory instead
-	{
-		t := &nodetasks.File{
-			Path:        "/var/log/kube-apiserver.log",
-			Contents:    fi.NewStringResource(""),
-			Type:        nodetasks.FileType_File,
-			Mode:        s("0400"),
-			IfNotExists: true,
-		}
-		c.AddTask(t)
-	}
+	c.AddTask(&nodetasks.File{
+		Path:        "/var/log/kube-apiserver.log",
```

Comment:
Could possibly make this path a constant as you're referring to it in a few places